analytics/src/test/java/com/segment/analytics/PropertiesTest.kt

```diff
@@ -0,0 +1,179 @@
+package com.segment.analytics
+
+import com.segment.analytics.Properties.Product
+import org.assertj.core.api.Assertions
+import org.junit.Before
+import org.junit.Test
+
+class PropertiesTest {
+    var properties: Properties? = null
+
+    @Before
+    fun setUp() {
+        properties = Properties()
+    }
+
+    @Test
+    fun revenue() {
+        properties?.putRevenue(3.14)
```

Comment:
is there a way to get rid of these `?` since we know in this test scenario that Properties is never null?